packages/attestation-service/src/sms/twilio.ts

```diff
@@ -76,15 +119,76 @@ export class TwilioSmsProvider extends SmsProvider {
     } catch (error) {
       throw new Error(`Twilio Messaging Service could not be fetched: ${error}`)
     }
+    if (this.verifyServiceSid) {
```

Comment:
Agreed. I'd like to leave both in since it gives us the validators the ability to quickly disable the Verify API option if needed. If we don't see any issues we can remove it in the next version.